site/src/analytics-os-boards.ts

```diff
@@ -0,0 +1,71 @@
+import { customElement, html, LitElement, property } from "lit-element";
+import { AnalyticsDataCurrent } from "../../worker/src/data";
+import "./components/analytics-chart";
+
+const friendlyBoardName: Record<string, string> = {
+  "intel-nuc": "Intel NUC",
+  "generic-x86-64": "Generic x86-64",
+  ova: "OVA",
+  "odroid-c2": "ODROID C2",
+  "odroid-c4": "ODROID C4",
+  "odroid-n2": "ODROID N2",
+  "odroid-xu4": "ODROID XU4",
+  rpi: "Raspberry Pi",
+  rpi0: "Raspberry Pi Zero",
+  rpi2: "Raspberry Pi 2",
+  rpi3: "Raspberry Pi 3",
+  rpi4: "Raspberry Pi 4",
+  tinker: "Asus TinkerBoard",
```

Comment:
```suggestion
  tinker: "ASUS Tinker Board",
```
Nit... That is how they stylize it on their website at least: https://tinker-board.asus.com/product/tinker-board.html